src/System.Private.CoreLib/src/System/Reflection/Emit/SymbolMethod.cs

```diff
@@ -118,6 +118,7 @@ public override RuntimeMethodHandle MethodHandle
         #endregion
 
         #region MethodInfo Overrides
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
         public override Type? ReturnType
         {
             get
```

Comment:
After the changes that were made to ReturnType elsewhere, this one can still be null?

Reply:
Yes I think if we have SymbolMethod that can return null, we should make the base nullable. I asked the same question in another suggestion from @jkotas to make `ReturnParameter` non-null but there is also an override which just returns null.

Reply:
The SymbolMethod constructor should replace `null` with `typeof(void)` - just like the MethodBuilder constructor.

Reply:
Fixed.